### DISCUSSION TITLE
Little more screenspace for event names

### DISCUSSION THREAD

**Discussion by munklarsen** [jhuckaby/Cronicle#605] [Ideas]
Hi,

First off - again, many thanks for this awesome tool. Well past 30K event/day now and still adding. 

Secondly. I have a "quality of life" ask of you. We use rather long names for our tasks which means at some point the names will be cut off and replaced by "...". We have large screens so real estate to show the names are there. Would there be any way to add some more pixels/characters to the names? Either dynamically or staticly (either through a global definition or a per use setting)? 

Added a picture to show it. Blurred it a bit to not give away what is running but it still should be clear that there is plenty of room. 

![image](https://github.com/jhuckaby/Cronicle/assets/121776852/daa1640f-7e38-4a10-8a93-0a291ebb91a7)

**Comment by dheise-rheinenergie-trading-com** [comment]
We have the same problem. Our workaround is a small bookmarklet that removes the max-width property from the column 'event name':

javascript code:
for (l of document.getElementsByClassName('ellip')) { l.style.maxWidth=''; }

converted to a bookmarklet:
javascript:(function()%7Bfor%20(l%20of%20document.getElementsByClassName('ellip'))%20%7B%20l.style.maxWidth%3D''%3B%20%7D%7D)()%3B

**Comment by jhuckaby** [comment]
Moved to issue: https://github.com/jhuckaby/Cronicle/issues/607

**Reply by munklarsen** [reply]
Dude, amazing! Thank you. We just updated and it looks much better :) Quality of life improvement right there! 

Any way to contribute financially / pay for some beers?

**Reply by jhuckaby** [reply]
You're welcome 😊 

I'll definitely have some monetization options in Cronicle v2, when it comes out 😉 .  Thank you for the offer!
